tests/SingleEventsTest.php

```diff
@@ -0,0 +1,452 @@
+<?php
+
+namespace Churchtools\Tests\ICal;
+
+use ICal\ICal;
+use PHPUnit\Framework\TestCase;
+
+class SingleEventsTest extends TestCase {
+
+    public function testFullDayTimeZoneBerlin() {
+        $checks = [
+            ['index' => 0, 'dateString' => '20000301', 'message' => '1st event, CET: '],
+        ];
```

Comment:
Should this use the old array notation as the library supports PHP 5.3?

Reply:
fixed.